src/fswAlgorithms/attControl/thrMomentumManagementCpp/_UnitTest/test_thrMomentumManagementCpp.py

```diff
@@ -99,6 +99,76 @@ def test_thrMomentumManagement(show_plots, hsMinCheck):
     np.testing.assert_allclose(trueVector, dataLog.torqueRequestBody, rtol=accuracy)
 
 
+@pytest.mark.parametrize("hd_B", [np.array([1.0, 0.0, 0.0])])
+def test_momentumBias(show_plots, hd_B):
+    unitTaskName = "unitTask"               # arbitrary name (don't change)
+    unitProcessName = "TestProcess"         # arbitrary name (don't change)
+
+    # Create a sim module as an empty container
+    unitTestSim = SimulationBaseClass.SimBaseClass()
+
+    # Create test thread
+    testProcessRate = macros.sec2nano(0.5)     # update process rate update time
+    testProc = unitTestSim.CreateNewProcess(unitProcessName)
+    testProc.addTask(unitTestSim.CreateNewTask(unitTaskName, testProcessRate))
+
+    # Construct algorithm and associated C++ container
+    module = thrMomentumManagementCpp.ThrMomentumManagementCpp()
+    module.ModelTag = "thrMomentumManagement"
+
+    # Add test module to runtime call list
+    unitTestSim.AddModelToTask(unitTaskName, module)
+
+    module.hd_B = hd_B  # Nms
+
+    # wheelSpeeds Message
+    rwSpeedMessage = messaging.RWSpeedMsgPayload()
+    rwSpeedMessage.wheelSpeeds = [10.0, -25.0, 50.0, 100.]
+    rwSpeedInMsg = messaging.RWSpeedMsg().write(rwSpeedMessage)
+
+    # wheelConfigData Message
+    fswSetupRW.clearSetup()
+    Js = 0.1
+    Gs = np.array([[1.0, 0.0, 0.0],
+                   [0.0, 1.0, 0.0],
+                   [0.0, 0.0, 1.0],
+                   [0.5773502691896258, 0.5773502691896258, 0.5773502691896258]])
+    fswSetupRW.create(Gs[0, :], Js)
+    fswSetupRW.create(Gs[1, :], Js)
+    fswSetupRW.create(Gs[2, :], Js)
+    fswSetupRW.create(Gs[3, :], Js)
+    rwConfigInMsg = fswSetupRW.writeConfigMessage()
+
+    # Setup logging on the test module output message so that we get all the writes to it
+    dataLog = module.deltaHOutMsg.recorder()
+    unitTestSim.AddModelToTask(unitTaskName, dataLog)
+
+    # setup message connections
+    module.rwSpeedsInMsg.subscribeTo(rwSpeedInMsg)
+    module.rwConfigDataInMsg.subscribeTo(rwConfigInMsg)
+
+    # Need to call the self-init and cross-init methods
+    unitTestSim.InitializeSimulation()
+
+    # Set the simulation time
+    unitTestSim.ConfigureStopTime(macros.sec2nano(0.5))  # seconds to stop simulation
+
+    # Begin the simulation time run set above
+    unitTestSim.ExecuteSimulation()
+
+    # Compute the wheel momentum vector
+    hs_B = np.zeros(3)
+    for i in range(4):
+        hs_B += Js * rwSpeedMessage.wheelSpeeds[i] * Gs[i, :]
+
+    # set the filtered output truth states
+    trueVector = hd_B - hs_B
+
+    # compare the module results to the truth values
+    accuracy = 1e-12
```

Comment:
:small_red_triangle_down: You can just set this explicitly in the assert call.

Reply:
Done.